encord/orm/dataset.py

```diff
@@ -1008,6 +1026,9 @@ class DatasetDataLongPolling(BaseDTO):
     errors: List[str]
     """Stringified list of exceptions."""
 
+    data_units_errors: Optional[List[DataUnitError]] = None
```

Comment:
This is optional mostly "just in case"; the "case" being "we released the SDK but had to roll back the BE so the field is not returned anymore".

We can probably make this type a notch nicer if we wait with merging this until after the BE is firmly out there.

Reply:
yeah, i'd vote for this option!